### DISCUSSION TITLE
How do I delete a training data to re-train?

### DISCUSSION THREAD

**Discussion by Tolulade-A** [vanna-ai/vanna#497] [Q&A]
I get this error when I try to delete one training data in the table `Exception: Error removing training data: You don't own this model and so cannot remove training data` .

How do I delete the entire training data to re-train especially as I am experimenting with Vanna, and would like to try different outcomes?

**Comment by anthonydu** [comment]
I believe `vn.remove_collection` is for that exact purpose. Correct me if I'm wrong.

**Reply by Tolulade-A** [reply]
okay, thanks
